dts/bindings/power/st,stm32wb0-pwr.yaml

```diff
@@ -0,0 +1,130 @@
+# Copyright (c) 2024 STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+description: STM32WB0 power controller
+
+compatible: "st,stm32wb0-pwr"
+
+include: "st,stm32-pwr.yaml"
+
+properties:
+  smps-mode:
+    type: string
+    required: true
+    description: |
+      SMPS mode selection
+
+      OFF:
+        - SMPS converter disabled
+        - LDOs supply voltage: VDD
+
+      WARNING: The SMPS must not be disabled on board using the
+      'SMPS supply configuration', so this mode should NEVER be
+      selected on these boards. Only use this mode if your board
+      is using the 'NOSMPS supply configuration'.
+
+      Refer to RM0505 §5.5 "SMPS step down regulator" for details.
+
+      PRECHARGE: (also called BYPASS)
+        - SMPS converter enabled - clock disabled
+        - LDOs supply voltage: VDD (though SMPS)
+          - Supplied current limitation can be programmed
+
+      RUN: (also called ON)
+        - SMPS converter enabled - clock enabled
+        - LDOs supply voltage: regulated SMPS output
+          - Target output voltage can be programmed
+    enum:
+      - "OFF"
+      - "PRECHARGE"
+      - "RUN"
+
+  smps-clock-prescaler:
+    type: int
+    default: 4
+    description: |
+      SMPS clock prescaler factor
+
+      The SMPS clock, CLK_SMPS, comes from a 16 MHz source that
+      goes through one of two prescalers, with a respective
+      division factor of 2 and 4. This property selects which
+      prescaler should be used.
+
+      Setting this property to 2 results in CLK_SMPS = 8 MHz.
+      Setting this property to 4 results in CLK_SMPS = 4 MHz.
+
+      All features of the SMPS can be used regardless of which
+      prescaler has been chosen. Since a slower clock results
+      in less power consumption, this property defaults to 4.
+
+      This property is not valid if `smps-mode` is "OFF".
+    enum:
+      - 2
+      - 4
+
+  smps-bom:
+    type: int
+    description: |
+      SMPS L/C BOM selection
+
+      Indicates which L/C BOM is present on the board.
+      Refer to RM0505 §5.5 for more details.
+
+      This property is not valid if `smps-mode` if "OFF",
+      and required otherwise.
+    enum:
+      - 1 # 1.5µH inductance, 2.2µF output capacitance
+      - 2 # 2.2µH inductance, 4.7µF output capacitance
+      - 3 #  10µH inductance, 4.7µF output capacitance
```

Comment:
Comments are not rendered in binding generated documentation. Please put these in description section.